### PR TITLE
import contacts

### PR DESCRIPTION
**Pull request by SergeyOdnosteblets**

### CODE REVIEW

File: src/pages/ListContainer/ListContainer.tsx [hunk 1]
```diff
@@ -57,6 +60,36 @@ export const ListContainer: React.FC<ListTypes> = ({ list, setList }) => {
           />
         )}
       </div>
+      <div>
```
Comment (t-shevchuk) [inline src/pages/ListContainer/ListContainer.tsx:71]: do we really need this div?

File: src/pages/ListContainer/ListContainer.tsx [hunk 2]
```diff
@@ -57,6 +60,36 @@ export const ListContainer: React.FC<ListTypes> = ({ list, setList }) => {
...
+                  id: item[0],
+                  firstName: item[1],
+                  lastName: item[2],
+                  age: item[3],
+                  country: item[4],
+                  gender: item[5],
+                };
+              });
+              users.shift();
+              setFilteredUsers(users);
```
Comment (t-shevchuk) [inline src/pages/ListContainer/ListContainer.tsx:88]: why?

File: src/pages/ListContainer/ListContainer.tsx [hunk 3]
```diff
@@ -57,6 +60,36 @@ export const ListContainer: React.FC<ListTypes> = ({ list, setList }) => {
           />
         )}
       </div>
+      <div>
+        <CSVReader
+          onUploadAccepted={(results: any) => {
+            if (results.errors.length) {
+              alert('faildToAddCount');
```
Comment (t-shevchuk) [inline src/pages/ListContainer/ListContainer.tsx:75]: typo `failed`

File: src/pages/ListContainer/ListContainer.tsx (hunk 2 above)
Comment (t-shevchuk) [reply src/pages/ListContainer/ListContainer.tsx:88]: filters should be reseted

File: src/pages/ListContainer/ListContainer.tsx [hunk 4]
```diff
@@ -57,6 +60,36 @@ export const ListContainer: React.FC<ListTypes> = ({ list, setList }) => {
...
+                  gender: item[5],
+                };
+              });
+              users.shift();
+              setFilteredUsers(users);
+              alert('addedCount');
+            }
+          }}>
+          {({ getRootProps }: any) => (
+            <div>
```
Comment (t-shevchuk) [inline src/pages/ListContainer/ListContainer.tsx]: remove

File: src/pages/ListContainer/ListContainer.tsx (hunk 1 above)
Comment (SergeyOdnosteblets) [reply src/pages/ListContainer/ListContainer.tsx:71]: changed

File: src/pages/ListContainer/ListContainer.tsx (hunk 2 above)
Comment (SergeyOdnosteblets) [reply src/pages/ListContainer/ListContainer.tsx:88]: filters reseted

**Pull request merged**